libr/io/section.c

```diff
@@ -57,11 +57,12 @@ static RIOSection *findMatching (RIO *io, ut64 paddr, ut64 vaddr, ut64 size, ut6
 R_API RIOSection *r_io_section_add(RIO *io, ut64 offset, ut64 vaddr, ut64 size, ut64 vsize, int rwx, const char *name, ut32 bin_id, int fd) {
 	int update = 0;
 	RIOSection *s;
-	if (size == 0 || size > 0xf0000000) {
-		if (size > 0 && size != UT64_MAX && size != UT32_MAX)
+	if (!size || size == UT64_MAX || size == UT32_MAX) { //hacky things which might give bad output in case size == UT32_MAX for 64bit elf. Check on basis of size, offset and file size would be a good idea.
```

Comment:
@alvarofe for the comment: https://github.com/radare/radare2/pull/5242#discussion_r69490005 , I have updated the condition. The eprintf was supposed to run when size is greater than 0xf0000000 but less than UT64_MAX and UT32_MAX. Now since `size > 0xf0000000` is no longer checked, I just removed the eprintf part and updated the main condition to fail in case of either of above condition.